src/components/SectionWrapper/SectionWrapper.tsx

```diff
@@ -0,0 +1,52 @@
+/** @jsxImportSource preact */
+import type { ComponentChildren, Key } from "preact"
+import { createElement } from "preact"
+import { useEffect, useRef } from "preact/hooks"
+import { updateTableOfContents } from "~/components/TableOfContents/tocStore"
+import GithubSlugger from "github-slugger"
+
+type Props = {
+  title: string
+  depth: number // Depth must be between 2 and 4 inclusive
+  children: ComponentChildren
+  idOverride?: string
+  updateTOC?: boolean
+  key?: Key
+}
+
+/**
+ * Use this component to wrap client-side components which feature section headers between h2 - h4.
+ * Provides sticky functionality, anchor tags and updates table of contents as needed
+ */
+function SectionWrapper({ title, depth, children, idOverride, updateTOC = true, key }: Props) {
```

Comment:
Suggestion: consider having this false by default. Currently when this wrapper is implemented, only one time this is false, needing `updateTOC=false` the rest of the times.

Reply:
This is a valid point as far as the current implementations go. However, it should be considered default behavior to update the table of contents when a new header is added after the first render, such as in the case of `FeedList.tsx` line 171. By changing the default to false, you risk future implementations where the TOC is not updated when it should be.